### PR TITLE
Add missing descriptions and leads

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: src/docs/content/guides/_index.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ---
-title: "Guides"
+title: Guides
+description: Detailed and practical advice
```
Comment (ccedric) [inline src/docs/content/guides/_index.md:3]: advice**s**

File: src/docs/content/reference/current/cheat-sheet.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Cheat-Sheet"
-description: ""
-lead: ""
+description: "Quick reference guides and quick reference sheets in one page"
```
Comment (ccedric) [inline src/docs/content/reference/current/cheat-sheet.md:3]: Quick reference guides and sheet in one page

File: src/docs/content/reference/current/extensions/_index.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
 title: "Extensions"
+description: "Third parties plugins"
```
Comment (ccedric) [inline src/docs/content/reference/current/extensions/_index.md:3]: Third party plugin

File: src/docs/content/reference/current/migration/2.1-to-2.2.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
----
+__---
```
Comment (ccedric) [inline src/docs/content/reference/current/migration/2.1-to-2.2.md:1]: I don't think that's intended

File: src/docs/content/reference/current/migration/_index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Migration"
+description: "What change between Gatling versions"
```
Comment (ccedric) [inline src/docs/content/reference/current/migration/_index.md:3]: What changes

File: src/docs/content/reference/current/project/_index.md [hunk 6]
```diff
@@ -1,3 +1,4 @@
 ---
 title: "Project Information"
+description: "Licenses, bugs tracking, contributors and much more."
```
Comment (ccedric) [inline src/docs/content/reference/current/project/_index.md:3]: bug tracking (with no s)

File: src/docs/content/reference/current/project/ci.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Continuous Integration"
-description: ""
-lead: ""
+description: "Travis and sonatype references"
```
Comment (ccedric) [inline src/docs/content/reference/current/project/ci.md:3]: I guess I'd put an uppercase on Sonatype?
(as Travis got one)

File: src/docs/content/guides/_index.md (hunk 1 above)
Comment (Isammoc) [reply src/docs/content/guides/_index.md:3]: NO!

advice is uncountable

https://www.wordreference.com/enfr/advice

File: src/docs/content/reference/current/extensions/_index.md (hunk 3 above)
Comment (Isammoc) [reply src/docs/content/reference/current/extensions/_index.md:3]: Third party plugins

File: src/docs/content/guides/_index.md (hunk 1 above)
Comment (ccedric) [reply src/docs/content/guides/_index.md:3]: oops, my bad

**Pull request closed without merging**